### PR TITLE
Move error div outside the dweet iframe

### PR DESCRIPTION
**Pull request by lionleaf**
Currently only works for the "new dweet" editing box.

screenshot 2017-03-13 00 04 38

### CODE REVIEW

**Comment by tbroadley** [comment]
Looks good to me :) Do you want to handle moving errors out of the iframe when remixing in this PR, or in a separate one?

File: dwitter/static/main.css [hunk 1]
```diff
@@ -53,6 +53,11 @@ input[type=submit].delete-button {
   font-family: 'Roboto', sans-serif;
 }
 
+.error-display {
+  color: red;
+  font-size: small;
+  font-family: sans-serif;
```
Comment (stianjensen) [inline dwitter/static/main.css]: Please use `font-family: 'Roboto', sans-serif;` like above, for consistency :)

Comment (lionleaf) [inline dwitter/static/main.css]: Fixed.

**Comment by lionleaf** [comment]
Now applies to all dweets!
I'll merge and deploy on review :)

**Comment by lionleaf** [comment]
ping

**Pull request merged**